### ISSUE TITLE
requesting HD4Free

### ISSUE THREAD

**Issue by porkie02** [cardigann/cardigann#333]
Could you be so kid and add https://hd4free.xyz if you need a invite to do it I can provide

**Comment by 2devnull** [comment]
+1

**Comment by porkie02** [comment]
Thats a shame I really like this site too.

**Comment by 2devnull** [comment]
Yes, indeed especially since Jackett is such a resource hog. I don't know why emby can run so well with Mono and Jackett with a simpler set of use-cases is such a resource hog.
